src/App.js

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react'
+import { Routes, Route, createSearchParams, useSearchParams, useNavigate } from "react-router-dom"
+import { useDispatch, useSelector } from 'react-redux'
+import 'reactjs-popup/dist/index.css'
+import { fetchMovies } from './data/moviesSlice'
+import { ENDPOINT_SEARCH, ENDPOINT_DISCOVER, ENDPOINT, API_KEY } from './constants'
+import Header from './components/Header'
+import Movies from './components/Movies'
+import Starred from './components/Starred'
+import WatchLater from './components/WatchLater'
+import YouTubePlayer from './components/YoutubePlayer'
+import './app.scss'
+
+const App = () => {
+
+  const state = useSelector((state) => state)
+  const { movies } = state  
```

Comment:
NIT
```suggestion
  const { movies }  = useSelector((state) => state)
```
You can use destructuring 😉